src/main/java/com/gluonhq/substrate/util/FileDeps.java

```diff
@@ -319,4 +387,67 @@ private void downloadJavaFXZip(String osarch, Path substratePath ) throws IOExce
         Logger.logDebug("Process zip javafx done");
     }
 
-}
+    /**
+     * Crafts Android SDK url and then downloads it 
+     * @throws IOException in case anything goes wrong.
+     */
+    private void downloadAndroidSdkZip() throws IOException {
+        Path sdk = configuration.getAndroidSdkPath();
+        String hostOs = configuration.getHostTriplet().getOs();
+        String androidSdkUrl = Strings.substitute(ANDROID_SDK_URL, Map.of("host", hostOs));
+        Logger.logInfo("Downloading Android SDK...");
+        FileOps.downloadAndUnzip(androidSdkUrl, sdk.getParent(), "android-sdk.zip", sdk.getFileName().toString(), "");
+        Logger.logInfo("Done");
```

Comment:
Maybe it can be removed. If not, use Logger.logDebug with a more descriptive message